### DISCUSSION TITLE
how to install to an existing partition

### DISCUSSION THREAD

**Discussion by dcihon** [ravynsoft/ravynos#374] [General]
How do I install to an existing partition.

**Comment by mszoek** [comment]
Have a look at [Installing ravynOS](https://wiki.ravynos.com/wiki/Installing_RavynOS) or read `/bin/install.sh` for the manual steps. Skip the steps where you create partitions and use your existing ones. You will still need an EFI partition and to copy the loader there. Swap is optional but recommended.
